xbmc/windowing/osx/WinEventsOSXImpl.mm

```diff
@@ -389,4 +409,52 @@ - (BOOL)SendXBMCMouseButtonEvent:(NSEvent*)nsEvent
   return location;
 }
 
+- (void)SendPlayerAction:(const int&)actionId
+{
+  CAction* action = new CAction(actionId);
+  CServiceBroker::GetAppMessenger()->PostMsg(TMSG_GUI_ACTION, WINDOW_INVALID, -1,
+                                             static_cast<void*>(action));
+}
+
+- (BOOL)HandleMediaKey:(int)keyCode
+{
+  BOOL passEvent = true;
```

Comment:
```suggestion
- (bool)HandleMediaKey:(int)keyCode
{
  bool passEvent = true;
```

also, maybe set it to false and set to true only in `default`?

Reply:
May I ask why bool instead of BOOL here?

Reply:
the calling code expects C++ bool, not sure why you want to use objc BOOL